### ISSUE TITLE
HTML lang attribute doesn't reflect language change

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1207]
Success Criterion 3.1.1 Language of Page  requires that the default human language of each web page can be programmatically determined.

When the site language is changed from English to Norwegian, the HTML lang attribute remainds `lang="en"` when it should be changed to `lang=no` (or perhaps `lang=nb`).
